tutorons/python/detect.py

```diff
@@ -0,0 +1,62 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+import logging
+import ast
+
+from tutorons.common.extractor import Region
+from tutorons.python.explain import explanations
+
+logging.basicConfig(level=logging.INFO, format="%(message)s")
+
+
+def filter_non_ascii(c):
```

Comment:
I think this code is also in `css/detect.py`.  If so, what would be the appropriate way to refactor the code so that we don't have this in two places?